Fix tests

jan-janssen opened the pull request:
At the moment, selective dynamics are not correctly read from the VASP output and also custom element types are not parsed correctly.

sudarsan1989 commented:
@jan-janssen , any idea why the tests seem to always fail for the separate branches but somehow it passes when it is merged with the master?

jan-janssen commented:
Because the branch no longer exists after merge

jan-janssen committed:
Fix tests

At the moment, selective dynamics are not correctly read from the VASP output and also custom element types are not parsed correctly.

jan-janssen committed:
black formatting

jan-janssen committed:
remove unused tests

jan-janssen committed:
do not report coverage
